draft-ietf-tls-tls13.md

```diff
@@ -2729,6 +2729,10 @@ TLS also allows other messages to be sent after the main handshake.
 These messages use a handshake content type and are encrypted under the application
 traffic key.
 
+Handshake messages sent after the handshake MUST NOT be interleaved with
+application data. That is, if a message is split over two or more handshake
+records, there MUST NOT be any application data records between them.
```

Comment:
I think we're in the "all alerts are fatal alerts" world, now, so isn't it illegal to send the other part of the handshake message after the alert?